tests/runtime/regression

```diff
@@ -108,3 +108,12 @@ NAME unaligned key with aligned value
 PROG BEGIN { @mapA["aaaabbb", 0] = 1; @mapA["ccccdddd", 0] = 1; }
 EXPECT Attaching 1 probe...
 TIMEOUT 1
+
+# In some cases, LLVM can generate code that violates the BPF verifier
+# constraint that the context pointer is not modified. #3629 introduces the use
+# of intrinsics that should ensure that this doesn't happen, but this runtime
+# test exercises a pattern that previously produced code that failed to verify.
```

Comment:
Nit: you could also add a comment to the original issue.

Reply:
Added link to both